roxctl/deployment/check/check.go

```diff
@@ -187,10 +203,18 @@ func (d *deploymentCheckCommand) Check() error {
 }
 
 func (d *deploymentCheckCommand) checkDeployment() error {
-	deploymentFileContents, err := os.ReadFile(d.file)
-	if err != nil {
-		return errors.Wrapf(err, "could not read deployment file: %q", d.file)
+	var deploymentFileContents []byte
+	for _, file := range d.files {
+		fileContents, err := os.ReadFile(file)
+		if err != nil {
+			return errors.Wrapf(err, "could not read deployment file: %q", file)
+		}
+		if len(deploymentFileContents) > 0 {
+			deploymentFileContents = append(deploymentFileContents, "\n---\n"...)
+		}
+		deploymentFileContents = append(deploymentFileContents, fileContents...)
 	}
+	log.Info("roxctl deployment check is handling file input of size %s", len(deploymentFileContents))
```

Comment:
A couple of things with this:
- you are using `Info` instead of `Infof`.
- you are using `%s` for a number, which should be `%d`.
- I don't really think the file size is a meaningful information to provide to the user. The file names could make sense, but other than that I'd rather drop it.

Reply:
Oops, I'll fix the `%d` and the `Infof`. In terms of usage, the idea behind logging this is a concern that if we append many deployments into one the resulting concatenation could become too large. By logging this we could stay on top of this. At least I think that's what the idea behind this was, thoughts @fredrb?

Reply:
The reason why I asked for this is because we have seen issues elsewhere where things fail due to large payloads and we can't really pinpoint which message was it. Logging the size of this file can help us in the future if users start complaining that deployment check is failing. 

It's a fair point that this information isn't meaningful to users, unless they want to open a ticket. 

Do we have debug logs in roxctl? If yes, perhaps let's add it as debug level. If users run into failures here we can ask them to run the command with debug logs enabled so we can see how large is the file roxctl is trying to send.